CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff

```diff
@@ -576,18 +604,22 @@
       </trans-unit>
       <trans-unit id="Find shared film credits" xml:space="preserve">
         <source>Find shared film credits</source>
+        <target>Znajdź wspólne udziały filmowe.</target>
```

Comment:
Brzmi trochę topornie, ale nie wiem jak ogarnąć to bezosobowo/zwięźle

Reply:
"Znajdź wspólne występy w filmach." ?